### DISCUSSION TITLE
Migrating node sync data to new machine

### DISCUSSION THREAD

**Discussion by damdinsharav** [Chia-Network/chia-blockchain#3185] [Q&A] [answered]
Hi, I need to migrate my node to new machine.

I know I could import my key and start syncing process. However, last time I did it took about 8 hours, not to mention now it's 3 fold bigger in terms network space.

I am wondering if there is faster way. Like copying certain folder from old machine to new machine?

**Comment by notpeter** [comment, marked as answer]
Stop chia on the new machine, and then copy: 
* `~/.chia/mainnet/db/blockchain_v1_mainnet.sqlite`
*  `~/.chia/mainnet/wallet/db/blockchain_wallet_v1_mainnet_*.sqlite`

**Reply by asztymelski** [reply]
does it works for you guys, ive tried and it failed, couldn't start my wallet

**Reply by r1235613** [reply]
> 
> 
> does it works for you guys, ive tried and it failed, couldn't start my wallet

You have to use `chia wallet show` and restore/generator your wallet.

**Reply by asztymelski** [reply]
tried several times and still issue 
`Connection error. Check if wallet is running at 9256
 Farming status: Syncing
 Total chia farmed: Unknown
 User transaction fees: Unknown
 Block rewards: Unknown
 Last height farmed: Unknown`
tried remove wallet, restore and so on and so forth

**Reply by UltimateBrent** [reply]
Just a note, I've done this a few times now. On the new machine, you need to enter your mnemonic etc. and start it so it'll generate the support files: `chia start farmer` then after a few seconds, stop it `chia stop farmer`, and you can copy those files in. Then just start it one last time and the node will be almost caught up (depending on when you copied from).

**Reply by asztymelski** [reply]
OK, then i screwed smth, cause im doing how you writing and sucks, so maybe I should try on "clean" machine, can I just remove `~/.chia/mainnet` and `~/chia-blockchain` directories to be sure is fresh? or you know places where might have some leftovers?
Many thanks for your help. This fckn slowness makes me crazy

**Reply by imangift** [reply]
You can download the Chia database here

https://github.com/chia-database

The download link is in the Radme file

**Reply by dideldei** [reply]
It is a dead link now.

**Reply by oleksiikoshlatyi** [reply]
[Here is the project](https://www.chiachain.link), that provides Chia full node database (blockchain_v1_mainnet.sqlite) via HTTP with a good bandwidth

**Comment by casa10** [comment]
you migrate your principal machine who was farmer,plotter, wallet to another ?
Because i would like to do that : my primary machine who is farmer, plotter, i would like to  put famer to another machine and my first only plot.
Look my schema.
I hope someone cal help me. 

![image](https://user-images.githubusercontent.com/8081413/116807136-b7c7bd80-ab31-11eb-8ebb-031ad7d9ebb5.png)

**Reply by damdinsharav** [reply]
@casa10 I am doing similar setup.
In your case you don't need to do anything on original machine(farmer,wallter).

1. Install chia on plotter with same key.
2. Start plotting and figure out ways to move plot files to hdd of original machine. There are many way. (rsync, or share from original machine via network)

**Reply by casa10** [reply]
Thank you, but i don't know if i explained well.
I have NAS to stock my plots.
My laptop is currently farmer and plotter .
I want him to become only plotter and to add new machine who will be the principal farmer and it will be able to plot too with this new one.
So is it same ?

**Reply by LuanPoi** [reply]
Yes it's the same. You can run only the plotter in your old machine by using the CLI with the command "chia plots create ....", in this way you will not start the farmer, and in the other machine you use normal.

**Reply by pongvech** [reply]
Your diagram is misleading. The laptop plotter need to be in the same position and add new desktop full node. In this case you just install chia on the desktop, disable uPnP, copy dbs from the laptop to the desktop, add plot folders(NAS). Once you are make sure wallet and full node are synced then you can stop the full node on laptop and use it just for plotting.

If you have port 8444 forwarded in the router you also need to forward that to the desktop IP address.
